packages/snaps-controllers/src/services/webview/WebViewMessageStream.test.ts

```diff
@@ -0,0 +1,71 @@
+import { sleep } from '@metamask/snaps-utils/test-utils';
+
+import { createWebViewObjects } from '../../test-utils/webview';
+import type { WebViewMessageStream } from './WebViewMessageStream';
+
+describe('WebViewMessageStream', () => {
+  let mockWebView: any;
```

Comment:
Let's stop defining these globally and use `createWebViewObjects` in each test. Each test should be able to run in isolation